scripts/build/Jenkins/Jenkinsfile

```diff
@@ -461,9 +457,26 @@ def UploadAPLogs(String platformName, String jobName, String workspace, Map para
     }
 }
 
-def PostBuildCommonSteps(String workspace, boolean mount = true) {
-    echo 'Starting post-build common steps...'
+def UploadTestArtifacts(String jobName, String workspace, String outputDirectory) {
+    catchError(message: "Error archiving test artifacts (this won't fail the build)", buildResult: 'UNSTABLE', stageResult: 'FAILURE') {
+        def cmakeBuildDir = [workspace, ENGINE_REPOSITORY_NAME, outputDirectory].join('/')
+        echo "Uploading Test Artifacts: ${cmakeBuildDir}/Testing"
+        ArchiveArtifactsOnS3("${cmakeBuildDir}/Testing", "test_artifacts/${jobName}", true)
+    }
+}
 
+def PostBuildCommonSteps(String workspace, Map params, boolean mount = true) {
+    echo 'Starting post-build common steps...'
+    if (params && params.containsKey('OUTPUT_DIRECTORY')){
```

Comment:
This can be simplified to `if (params.containsKey('OUTPUT_DIRECTORY'))`

Reply:
I think params can still be null here? I think there are some build_jobs where it will pass in a null param